tests/conftest.py

```diff
@@ -152,7 +159,7 @@ def factory(
             secret = secret or secret_factory(interface=interface)
 
             if interface == constants.CEPHBLOCKPOOL:
```

Comment:
This way it adds custom pool only to RBD. If you want it for CephFS too, you should remove interface condition and use something like:
```
block_pool = block_pool or ceph_pool_factory(interface=interface)
interface_name = block_pool.name
```
Also block_pool variable should be renamed to pool.

Reply:
done